packages/core/src/scss/components/checkout/checkoutRemote/_checkoutRemote.scss

```diff
@@ -15,6 +15,22 @@
     > .AmazonPayContainer {
         width: remCalc(200px);
     }
+
+    > #applepayCheckoutButton button,
+    > #braintreepaypalcreditCheckoutButton,
+    > #braintreepaypalCheckoutButton {
+        width: 160px;
+    }
```

Comment:
@animesh1987 It turns out that both PayPal buttons are 10px short, caused by PayPal SDK.
This is the result of applying the width override.
<img width="457" alt="Screenshot" src="https://user-images.githubusercontent.com/88361607/218591733-19476386-3029-4244-ba48-ec103f6c6a05.png">